### PR TITLE
add multiselect forms for requesting candidates

### PR DESCRIPTION
**Pull request by Dmitruk89**
1. add 4 multiselect fields for requesting candidates
Снимок экрана 2021-11-12 в 13 07 42
.

2. add base-URL into api-service request methods.

### CODE REVIEW

File: src/app/components/candidate-table/candidate-table/candidate-table.component.html [hunk 1]
```diff
@@ -1,10 +1,49 @@
-<div class="filter_container">
-    <app-multiselect-search [placeholder]="'select'" [data]="labels" class="multiselect"></app-multiselect-search>
+
```
Comment (mhasilau) [inline src/app/components/candidate-table/candidate-table/candidate-table.component.html:1]: the first string is empty

File: src/app/components/candidate-table/candidate-table/candidate-table.component.html [hunk 2]
```diff
@@ -1,10 +1,49 @@
...
+        </div>
+        <button class="btn filled-btn">Request</button>
+    </mat-expansion-panel>
     <mat-form-field appearance="standard">
         <mat-label>{{'candidateList.filter' | translate}}</mat-label>
         <input matInput (keyup)="applyFilter($event)" placeholder="{{'candidateList.example' | translate}}" #input>
     </mat-form-field>
-</div>
+
```
Comment (mhasilau) [inline src/app/components/candidate-table/candidate-table/candidate-table.component.html:47]: pls del one empty string

**Review by mhasilau — APPROVED** [review]

File: src/app/components/candidate-table/candidate-table/candidate-table.component.html (hunk 2 above)
Comment (Dmitruk89) [reply src/app/components/candidate-table/candidate-table/candidate-table.component.html:41]: fixed

File: src/app/components/candidate-table/candidate-table/candidate-table.component.html [hunk 3]
```diff
@@ -1,48 +1,42 @@
-
-    <mat-expansion-panel>
-        <mat-expansion-panel-header>
-          <mat-panel-title>
+<mat-expansion-panel>
+    <mat-expansion-panel-header>
+        <mat-panel-title>
             Candidate list request
```
Comment (mhasilau) [inline src/app/components/candidate-table/candidate-table/candidate-table.component.html:4]: we have only English language?

File: src/app/components/candidate-table/candidate-table/candidate-table.component.html [hunk 4]
```diff
@@ -1,48 +1,42 @@
...
-    <mat-form-field appearance="standard">
-        <mat-label>{{'candidateList.filter' | translate}}</mat-label>
-        <input matInput (keyup)="applyFilter($event)" placeholder="{{'candidateList.example' | translate}}" #input>
-    </mat-form-field>
+        </mat-panel-title>
+        <mat-icon>assignment</mat-icon>
+    </mat-expansion-panel-header>
+    <div class="inputs_container">
+        <mat-form-field appearance="fill">
+            <mat-label>Status</mat-label>
```
Comment (mhasilau) [inline src/app/components/candidate-table/candidate-table/candidate-table.component.html:10]: the same in lines 10, 16, 22, 28, 34

File: src/app/components/candidate-table/candidate-table/candidate-table.component.html (hunk 3 above)
Comment (vstrabko) [reply src/app/components/candidate-table/candidate-table/candidate-table.component.html:4]: +1

File: src/app/components/candidate-table/candidate-table/candidate-table.component.html (hunk 4 above)
Comment (vstrabko) [reply src/app/components/candidate-table/candidate-table/candidate-table.component.html:10]: +1

**Review by Nicolai-boiko — APPROVED** [review]

**Review by vstrabko — APPROVED** [review]

**Review by mhasilau — APPROVED** [review]

**Pull request merged**